app/src/main/res/values/userID.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="userID">t10p26</string>
```

Comment:
How was this not blocked by the `.gitignore`? Was branch created before `userID.xml` was put in the `.gitignore`, or was this manually added?